NL2go_Sync.xml

```diff
@@ -0,0 +1,105 @@
+<_>
+   <form_key>AbXjnhnkIK0Ye1JD</form_key>
+   <_create/>
```

Comment:
is this field required

Reply:
This is some magento1 crap. If I remove it, it will be created back the next time we generate a new package.